nestkernel/connection_manager.cpp

```diff
@@ -1562,11 +1565,19 @@ nest::ConnectionManager::resize_connections()
 void
 nest::ConnectionManager::sync_has_primary_connections()
 {
+  has_primary_connections_ = check_primary_connections_.any_true();
+#pragma omp barrier
```

Comment:
is this in a threaded part?

Reply:
It was, I have reverted back to the original, as that might be easier to read.